### DISCUSSION TITLE
Photos missing from Photoprism

### DISCUSSION THREAD

**Discussion by baurmatt** [photoprism/photoprism#2931] [Q&A]
#### 1. What is not working as documented?

We found, that certain pictures are missing from Photoprism. Two examples:

```
root@nas01:/storage/photoprism/data# ls -la ./originals/Photos/2022/07/2022-07-02_19-32-26_IMG_1650.HEIC
-rw-rw-r-- 1 s0me0ne s0me0ne 911609 Nov  2 18:22 ./originals/Photos/2022/07/2022-07-02_19-32-26_IMG_1650.HEIC
root@nas01:/storage/photoprism/data# ls -la ./originals/Photos/2022/07/2022-07-02_17-32-26.HEIC
-rw-rw-r-- 1 s0me0ne s0me0ne 3170820 Nov  2 18:29 ./originals/Photos/2022/07/2022-07-02_17-32-26.HEIC
```

Those files have been synced to Photoprism from WebDAV. I've ran the following commands to ensure all files have been properly indexed:

* `photoprism convert -f`
* `photoprism thumbs -f`
* `photoprism index -f` 

#### 2. How can we reproduce it?

?

#### 3. What behavior do you expect?

All files on disk are shown in Photoprism.

#### 4. What could be the cause of your problem?

It's unclear to me why this happens. Strangely other HEIC images are properly shown. Even ones that are taken within the same _minute_ with the same phone.

#### 5. Can you provide us with example files for testing, error logs, or screenshots?

I'd prefer not to, because those files contain private pictures of children.

#### 6. Which software versions do you use?

(a) PhotoPrism Architecture & Build Number: AMD64 221103

(b) Database Type & Version: MariaDB

(c) Operating System Types & Versions: Linux (Host), Linux/Mac (Client)

(d) Browser Types & Versions: Firefox

(e) Ad Blockers, Browser Plugins, and/or Firewall Software? All of them

#### 7. On what kind of device is PhotoPrism installed?

(a) Device / Processor Type: [ODROID-H2](https://www.hardkernel.com/shop/odroid-h2/), Intel J4105

(b) Physical Memory & Swap Space in GB: 16GB RAM, 1GB SWAP

(c) Storage Type: SSD

(d) Anything else that might be helpful to know?

#### 8. Do you use a Reverse Proxy, Firewall, VPN, or CDN?

traefik:v2.8.5

**Comment by baurmatt** [comment]
Some more things I've checked:

* They're not listed as hidden or error
* They're not listed in review
* Even though other images are quite similar and taken in the same minute, I can't find them as "stacked"

**Comment by lastzero** [comment]
Before you report a bug, please go through the checklist at https://docs.photoprism.app/getting-started/troubleshooting/#missing-pictures and ensure this is reproducable, either with pictures you send use privately or any other samples. We cannot help you otherwise. Thank you! 💎

**Reply by lastzero** [reply]
My guess is you need to fix filesystem permissions.

**Comment by baurmatt** [comment]
I've found it... It's stacked to other pictures. As the primary picture is a live picture, the symbol in the upper left corner is live picture symbol, not the stacked one. That's why I didn't looked at it first. I guess it would be helpful to prioritize the stack symbol for stacks or even show all relevant symbols for a stack :)

**Reply by lastzero** [reply]
No space in the mosaic view for multiple icons.

**Reply by baurmatt** [reply]
I'm unsure what exactly the "mosaic view" is, but the space here looks quite promising :)

![image](https://user-images.githubusercontent.com/2098878/203166242-36f9d262-5462-4f9e-8f48-3fc4b88658c9.png)

(Different picture, same problem :))

Would you be willing to accept an issue for this? What ever the solution look like in the end, I think this should be solved because it's very confusing for the user.

**Reply by lastzero** [reply]
![Screenshot_20221121_230432](https://user-images.githubusercontent.com/301686/203167346-b1b99b15-72c1-42db-a451-41e3b9adbccb.jpg)

This is the mosaic view. It's actually the first time someone is suggesting such changes, so I guess we'll keep it like it is, also given the many many other feature requests.

**Reply by baurmatt** [reply]
Ah, got it! I've only looked at it from the laptop browser sight, but (obviously in 2022) Photoprism uses responsive design to scale it to different devices and the space on a phone looks very limited indeed.

Anyway, thanks for all your work on Photoprism and insights in this discussion! :)
